### ISSUE TITLE
Use json-seq for line-based json responses

### ISSUE THREAD

**Issue by gwire** [openrightsgroup/blocked-org-uk#409]
A request for `https://www.blocked.org.uk/stream-results?url=http%3A//www.example.com` will currently include the content declaration

```
Content-Type: text/html; charset=utf-8
```

but the result is actually multiple lines containing json objects.

Please use JSON text sequences, per [RFC 7464](https://datatracker.ietf.org/doc/html/rfc7464) with the content declaration

```
Content-Type: application/json-seq
```

Also, see [this note](https://github.com/openrightsgroup/blocked-org-uk/blob/3fa1e28cd84212a7bea3af8042c5cb6ed757803f/modx/snippets/CMPMiddleware/APIHelper#L11).

**Comment by dantheta** [comment]
Good catch.  The stream-results response isn't fully compliant with the RFC (no ascii record separator, only newlines).  I've set it to application/json for now, and when I have more time to test I'll update the parser and mimetype to json-seq.
